src/USVPlugin.cpp

```diff
@@ -31,6 +31,7 @@ void USVPlugin::Load(physics::ModelPtr _model, sdf::ElementPtr _sdf)
     mNode->Init();
 
     mActuators = new Actuators(mModel, mNode);
+    mThrusters = new Thrusters;
```

Comment:
For some reason, I can't change where `mThrusters` is initialized. When I moved this operation to be inside `loadThrusters`, rock-gazebo would just close after setting up the listener for `wet_paint`:

```console
listening to gazebo:usis:wet_paint for wet_paint
gzserver finished (11064)
```